project.py

```diff
@@ -31,14 +31,28 @@ class BoardSearcher:
     last_saved_slide_mask = None
     last_saved_hash = None
     last_hash = None
+    similarity = 0.60
     hash_function = None
     debug = True
 
+    grid_size = (8, 8)
+    stored_section = None
+    last_saved_section = None
+    section_threshold = 15
+    reject_threshold = 10
+    section_overlap = 10
+
+    debug_image = None
+
     def __init__(self, n_slides=0,
                  frame_counter=0,
                  save_interval=30,
                  similarity=0.60,
                  compare_function='dhash',
+                 section_thresh=15,
+                 section_rjct=10,
```

Comment:
@Foxiy Are you OK with the name of this variable? I get what it means (now) but I am not sure if I would, had I not read the code in full.